au4/src/effects/builtin/tonegen/ToneView.qml

```diff
@@ -1,7 +1,129 @@
-import QtQuick 2.15
+import QtQuick
+import QtQuick.Controls
+import Muse.UiComponents
+import Audacity.Effects
+import Audacity.Playback
+import "../common"
 
-Rectangle {
+EffectBase {
 
-    color: "#009090"
+    property string title: "Tone" // review
```

Comment:
I think we can return it from the effect, through its view model... But I don't see that there is a translatable name for the user. So for now we can do it like this:
```
property string title: qsTrc("effects, "Tone")
```
and then we'll see how to improve it

Reply:
Thanks, I missed that!